packages/babel-plugin/src/index.ts

```diff
@@ -87,6 +87,31 @@ const getDebugId = (path: NodePath<t.CallExpression>) => {
     return;
   }
 
+  // Special case: Handle `export const [themeClass, vars] = createTheme({});`
+  // when it's already been compiled into this:
+  //
+  // var _createTheme = createTheme({}),
+  //   _createTheme2 = _slicedToArray(_createTheme, 2),
```

Comment:
Should we add any checking to see if this line is related to the `createTheme` call? Just want to be sure we don't get any false positives?

Reply:
Considered this but thought it's probably overkill given it's already constrained to variable declarations defining 4 variables where the first one calls `createTheme` — something that effectively never happens naturally. How strongly do you feel on this one?

Reply:
Not strongly 👍